storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -154,6 +154,61 @@ Options
    :option:`xtrabackup --defaults-group` option. It is needed for
    ``mysqld_multi`` deployments.
 
+.. option::  --dump-innodb-buffer-pool
+
+   This option controls whether or not a new dump of buffer pool
+   content should be done.
+
+   If this option is set to `ON`, the file :file:`ib_buffer_pool` is copied as
```

Comment:
when `--dump-innodb-buffer-pool` xtrabackup asks the server to start buffer pool dump (it takes some time to complete and is done in background) at the beginning of a backup.